TeamSnapSDK/SDK/DataTypes/TSDKUser.m

```diff
@@ -29,6 +29,18 @@ + (NSString *)SDKType {
     return @"user";
 }
 
+- (NSString *)fullName {
+    if ((self.firstName && self.firstName.length>0) && (self.lastName && self.lastName.length>0)) {
```

Comment:
self.firstName && self.firstname.length is redundant. self.firstname.length will have the same result when self.firstname is nil.